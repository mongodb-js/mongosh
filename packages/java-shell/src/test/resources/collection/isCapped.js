// before
db.coll.deleteMany({});
db.coll.insertOne({a: 1});
// command
wasCapped = db.coll.isCapped()
// command
db.coll.convertToCapped(10240)
// command
isCapped = db.coll.isCapped()
// clear
db.coll.drop();
