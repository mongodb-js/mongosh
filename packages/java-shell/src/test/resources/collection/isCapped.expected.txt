false
{ "ok": 1 }
true
